<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="axesColorLabel">
       <property name="text">
        <string>&amp;Axes Color</string>
       </property>
       <property name="buddy">
        <cstring>axesColorButton</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="axesColorButton"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="samplesColorLabel">
       <property name="text">
        <string>&amp;Samples Color</string>
       </property>
       <property name="buddy">
        <cstring>samplesColorButton</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="samplesColorButton"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="drawLinesCheckBox">
       <property name="text">
        <string>&amp;Draw Lines Between Samples</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
